components/gfx/display_list/mod.rs

```diff
@@ -184,14 +184,11 @@ impl DisplayList {
 
         doit(&(self.all_display_items()));
         if self.children.len() != 0 {
-            println!("{} Children stacking contexts list length: {}", indentation, self.children.len());
-            for subitem in self.children.iter() {
-                doit(&subitem.display_list.all_display_items());
-                if subitem.display_list.children.len() != 0 {
-                    // Rant: String doesn't have a substr() method that won't overflow if the
-                    // selected range is bigger than the string length.
-                    subitem.display_list.print_items(indentation.clone()+&indentation[0..min_length]);
-                }
+            println!("{} Children stacking contexts list length: {}",
+                     indentation,
+                     self.children.len());
```

Comment:
Yeah, it is, because as it is today children have their items printed twice: once during the invocation of `doit` and the second time during the recursive `print_items` call.